src/main/webapp/styles/recommendation-map.css

```diff
@@ -1,3 +1,18 @@
+h2 {
+  color: #000;
+}
+
+h3 {
+  color: #000;
+}
+
+p {
+  color: #000;
+}
+
+label {
+  color: #000;
+}
```

Comment:
Nit: Could you try writing this? 
```
h2, h3, p, label {
  color: black;
}
```
It should work if I am not wrong!

Also, using color name when you can makes it more readable.